pkg/textparse/openmetricsparse.go

```diff
@@ -170,6 +182,34 @@ func (p *OpenMetricsParser) Metric(l *labels.Labels) string {
 	return s
 }
 
+// Exemplar writes the exemplar of the current sample into the passed
+// exemplar. It returns the string from which the metric was parsed.
+func (p *OpenMetricsParser) Exemplar(e *exemplar.Exemplar) string {
```

Comment:
How do you distinguish between a series with no exemplar, and one with no labels and a value of 0?